code/mob/living.dm

```diff
@@ -1104,10 +1104,16 @@
 		else
 			maptext_color = src.last_chat_color
 
+		var/popup_style = src.speechpopupstyle
+
+		if (src.find_type_in_hand(/obj/item/megaphone))
+			var/obj/item/megaphone/megaphone = src.find_type_in_hand(/obj/item/megaphone)
+			popup_style = "font-size: [megaphone.maptext_size]px;"
+
```

Comment:
Should this be `+=` otherwise it will replace the bioeffects?
/datum/bioEffect/speech/comic